.github/workflows/WE2E_tests.yaml

```diff
@@ -0,0 +1,92 @@
+# This workflow runs the CI for the Regional Workflow End to End Tests
+name: Run WE2E Tests
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches:
+      - rrfs_ci
+    types: 
+      - labeled
+
+# Use custom shell with -l so .bash_profile is sourced which loads intel/oneapi/setvars.sh, and lmod path 
+defaults:
+  run:
+    shell: bash -l {0}
+
+jobs:
+  
+  # The run_id is only available in this worfklow if the workflow which builds the SRWA in build_SRWA.yml was successful.
+  Configure_and_Submit_WE2E_tests:
+    name: Configure matrix, 
+    runs-on: self-hosted
+    if: ${{ github.event.label.name == 'ci-aws-intel-WE' }}
+    outputs: 
+      run-id: ${{ steps.get_run_id.outputs.run-id }}
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+
+    steps:
+      - name: Fail safe for SRW App Build Failure, and set the run_id for use in path of SRWA build 
+        id: get_run_id
+        run: |
+          FILE="${{ github.workspace }}/run_id.txt"
+          if [[ -f "$FILE"]]; then 
+            echo "::set-output name=run-id::$(cat ${{ github.workspace}}/run_id.txt)"
+          else 
+            echo "SRWA Build failed"
+            exit 1 
+          fi
+
+      - name: Set matrix of tests from machine_suites/ci.txt
+        id: set-matrix
+        run: | 
+          echo "::set-output name=matrix::$(cat ufs-srweather-app-${{ steps.get-run-id.outputs.run-id }}/regional_workflow/tests/WE2E/machine_suites/ci.txt | jq -R -s -c 'split("\n")[:-1]')"
+          echo 
+
+      - name: Create experiment directory
+        run: |
+          mkdir -p /scratch1/ci-WE2E-test/${{ github.run_id }}
+
+      - name: Submit all experiments (WE2E tests) 
+        run: | 
+          cd ufs-srweather-app-${{ steps.get-run-id.outputs.run-id }}/regional_workflow/tests/WE2E/
+          module use -a /scratch1/miniconda3/contrib_miniconda3/modulefiles
+          module load miniconda3/4.11.0
+          conda activate regional_workflow
+          ./run_WE2E_tests.sh \
+            tests_file=machine_suites/ci.txt \
+            machine=aws \
+            account=account \
+            expt_basedir=/scratch1/ci-WE2E-test/${{ github.run_id }} \
+            exec_subdir=bin_intel/bin \
+            cron_relaunch_intvl_mnts=01 \
+            build_mod_fn=build_aws_intel
+
+      - name: Check the status of all experiments
+        run: |
+          echo "Check Workflow Generation/Status"
+          cd ufs-srweather-app-${{ steps.get-run-id.outputs.run-id }}/regional_workflow/tests/WE2E/
+          ./get_expts_status.sh expts_basedir=/scratch1/ci-WE2E-test/${{ github.run_id }}
+    
+  check-each-we2e-status: 
+    name: Check ${{ matrix.WE2Etest }} Status
```

Comment:
It doesn't look like the variable expansion works on the PR interface where it shows which tests pass/fail.  I recommend you just use a simple text description here along the lines of "Check Status of all WE2E runs"

Reply:
You're right, when the test was failing, the variable expansion wasn't working on the PR interface. It does work when viewing the Workflow Summary from the Action tab [(example)](https://github.com/NOAA-GSL/regional_workflow/actions/runs/2701589535). For the name of the job, I changed it to  "Check Status of all WE2E Runs".  It turns out the `${{ matrix.WE2Etest }}` variable is by default appended to the name of the job (when executing a job matrix), and is expanded in the PR interface & workflow run summary.